ReactCommon/cxxreact/NativeToJsBridge.h

```diff
@@ -100,7 +92,7 @@ class NativeToJsBridge {
   // will try to run the task on m_callback which will have been destroyed
   // within ~NativeToJsBridge(), thus causing a SIGSEGV.
   std::shared_ptr<bool> m_destroyed;
-  std::shared_ptr<react::ExecutorDelegate> m_delegate; // TODO(OSS Candidate ISS#2710739)
```

Comment:
I think the ExecutorDelegate stuff was added specifically for SandboxExecutor on Win32. I'm not sure if anything on Android started using this, but it may be removable.

I'm going to purge it from React Native Windows once I'm done with override tooling stuff. I can try to remove it here once that's done if Android isn't using it.

Reply:
Thanks .. I almost removed them first .. but later put it back to remve after further validation ..